tools/ctl/ctl.go

```diff
@@ -216,6 +217,94 @@ var getResults = &cobra.Command{
 	},
 }
 
+var missingAttestations = &cobra.Command{
+	Use:   "missing -project <ID> -run <ID> -attestation-bucket <name> -format <list|bench>",
```

Comment:
nit: maybe `find-missing`? or `find-missing-attestations`?

Reply:
I had missing-attestations originally but I figured that was too long. I can change it back to `find-missing-attestations`